src/include/rados/rgw_file.h

```diff
@@ -91,6 +91,7 @@ void rgwfile_version(int *major, int *minor, int *extra);
 */
 #define RGW_LOOKUP_FLAG_NONE    0x0000
 #define RGW_LOOKUP_FLAG_CREATE  0x0001
+#define RGW_LOOKUP_FLAG_RCB     0x0002
```

Comment:
the meaning of `RCB` is not immediately obvious here, a comment would be helpful